spec/features/home_spec.rb

```diff
@@ -93,14 +103,53 @@ def login_user
 
     it "selects only relevant cards to display" do
       @test_card = FactoryGirl.create :card, review_date: Date.tomorrow,
+                                             deck_id: deck.id,
                                              user_id: user.id
       expect(page).to have_content "Нет карточек для повторения"
     end
 
     it "Doesn`t show cards of other user" do
       @test_card = FactoryGirl.create :card, review_date: Date.today,
+                                             deck_id: deck.id,
                                              user_id: second_user.id
       expect(page).to have_content "Нет карточек для повторения"
     end
   end
+
+  describe "card selection process with current deck chosen" do
+    let(:alter_deck) { FactoryGirl.create :deck, title: "alter_title",
+                                                 user_id: user.id 
+                     }
+
+    it "displays No Card message when there are no cards in current deck and \
+    there are some in other deck" do
+      
+      @test_deck = FactoryGirl.create :deck, title: "Title", 
+                                             user_id: user.id
+      visit decks_path
+      click_link('Сделать текущей')
+      @test_card = FactoryGirl.create :card, original_text: "Правильное значение", 
+                                             review_date: Date.yesterday,
+                                             deck_id: alter_deck.id,
+                                             user_id: user.id
+      visit root_path
+
+      expect(page).to have_content "Нет карточек для повторения"
+    end
+
+    it "displays Card when there is smthg to display in current deck" do
+      
+      @test_deck = FactoryGirl.create :deck, title: "Title", 
+                                             user_id: user.id
+      visit decks_path
+      click_link('Сделать текущей')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.